### DISCUSSION TITLE
Need help figuring out how to best gracefully kill my sandbox workers after exiting out of nodemon

### DISCUSSION THREAD

**Discussion by brianzhou13** [taskforcesh/bullmq#2205] [Q&A]
At the moment, I'm switching between `nodemon` and `tsx` during development.

One issue I'm having is when I kill the nodemon/tsx watcher, I seem to get phantom processes lingering around, and I have to manually `kill -9 PID` each to bring my machine's CPU usage down.

How have people dealt with this issue during development?

Thank you!

**Comment by manast** [comment]
Have you tried this? https://docs.bullmq.io/guide/going-to-production#gracefully-shut-down-workers

**Comment by brianzhou13** [comment]
My apologies for the delay, I did! I do have that inserted.

So a bit about my setup, I have nodemon running and that spins up an express server (`nodemon ./src/index.ts`).

But whenever I kill it midway while there are tasks in the queue, I get this message:

<img width="650" alt="Screenshot 2023-10-11 at 11 47 30 PM" src="https://github.com/taskforcesh/bullmq/assets/5092263/e3ca87d8-220c-4087-9768-2feb29d44f52">

Then I check what processes are running, and I see what I assume are the node sandbox workers spun up from bullmq (??)

<img width="2232" alt="Screenshot 2023-10-11 at 11 48 06 PM" src="https://github.com/taskforcesh/bullmq/assets/5092263/8e1b29f6-73f0-4b3d-9662-b364d74389a9">

Note -- I am spinning up chromium in each worker to scrape a page. Could that be an issue?

Thanks again!
